src/org/omegat/core/dictionaries/LingvoDSL.java

```diff
@@ -66,7 +66,7 @@ public LingvoDSL(File file) {
                 }
                 s = RE_SKIP.matcher(s).replaceAll("");
                 if (Character.isWhitespace(s.codePointAt(0))) {
-                    trans.append(s).append('\n');
+                    trans.append(s.substring(1)).append('\n');
```

Comment:
You don't know how many `char`s that first codepoint is, so `substring(1)` is wrong. If your intent is to strip the first whitespace codepoint, it should be `s.substring(s.offsetByCodePoints(0, 1))`, or you could grab the first codepoint first as `int cp = s.codePointAt(0)` and then do `s.substring(Character.charCount(cp))`.

However it seems to me that if you just want to trim the whitespace you should just do `s.trim()`.